### DISCUSSION TITLE
Support daemon mode like eslint_d

### DISCUSSION THREAD

**Discussion by sys9kdr** [textlint/textlint#1286] [Ideas]
Thank you for your great project! I 

I propose adding a daemon mode similar to [eslint_d](https://github.com/mantoni/eslint_d.js/). This will keep the tool running in the background, leading to faster response times and improved performance.
I use textlint in my project. This erxecution speed is currently slow.
In my case, over 60s running `textlint`. I would like to run `textlint` faster.

# Benefits(maybe)

* Reduced startup time
* Improved performance
* Improved resource usage, especially when running lints as an editor tool (lints triggered on save)

**Comment by azu** [comment]
It is not technically difficult.
`textlint` package probably exposes the necessary API.

- [Use as Node Modules - textlint](https://textlint.github.io/docs/use-as-modules.html)

You can load rules with `loadTextlintrc()` and keep this in memory so you don't have to load rules on every startup. It can be created freely.

I don't have much motivation to create daemons like `eslint_d` as I don't have the opportunity to use them.
However, I am a bit interested in generic mechanisms such as the Language Server Protocol (LSP).
LSP may be a possible direction for [@textlint/editor](https://github.com/textlint/editor#textlinteditor).

**Reply by sys9kdr** [reply]
Thank you. I'll try to tackle this on my own. I'll explore the methods you taught me and consider incorporating libraries like https://github.com/mantoni/core_d.js. I'm going to close this discussion."
